### PR TITLE
1931 Canadian Census

### PR DESCRIPTION
**Pull request by manf0001**
The Canadian 1931 Census is digitized and starting to be seen online at Ancestry or the Canadian Archives for download.  Adding that year to webtrees.

### CODE REVIEW

File: app/Census/CensusOfCanada1931.php [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    {
+        return [
+new CensusColumnFullName($this, 'Name', 'Name of each person in family, household or institution'),
+            new CensusColumnNull($this, 'Place of Abode', '(In rural localities give parish or township. In cities, towns and villages, give street a>
+            new CensusColumnNull($this, 'Own/Rent', 'Home owned or rented'),
+            new CensusColumnNull($this, 'Value', 'If Owned give value. If rented, give rent paid per month'),
+            new CensusColumnNull($this, 'Class', 'Class of House: Apartment, Row or Terrace, Single house, Semi-Detached, Flat'),
+            new CensusColumnNull($this, 'Materials', 'Material of Construction: Stone, Brick, Wood, Brick Veneered, Stucco, Cement bricks'),
+            new CensusColumnNull($this, 'Rooms', 'Rooms occupied by this family'),
+            new CensusColumnNull($this, 'Radio', 'Has this fmily a radio?'),
```
Comment (fisharebest) [inline app/Census/CensusOfCanada1931.php:52]: Typo? Fmily => Family?

File: app/Census/CensusOfCanada1931.php [hunk 2]
```diff
@@ -0,0 +1,85 @@
+?php
```
Comment (fisharebest) [inline app/Census/CensusOfCanada1931.php:1]: Missing the opening `<` character

**Comment by fisharebest** [comment]
Could you also add/update the corresponding test scripts?
* `app/Census/CensusOfCanadaTest.php`
* `app/Census/CensusOfCanadaTest1931.php`

They should be pretty straightforward.  Just look at one of the other test scripts.

Thanks!

File: app/Census/CensusOfCanada1931.php (hunk 1 above)
Comment (manf0001) [reply app/Census/CensusOfCanada1931.php:52]: opps yes, a typo.

File: app/Census/CensusOfCanada1931.php (hunk 2 above)
Comment (manf0001) [reply app/Census/CensusOfCanada1931.php]: Sorry about that, I didn't notice it was missing when I copied and past it into github.  I've added it in now.

**Comment by manf0001** [comment]
sorry, I didn't know about the test scripts.. will work on that, then uploaded and issue a PR for it ASAP

**Comment by fisharebest** [comment]
Thanks!  This will be included in the next release.

**Pull request merged**